### PR TITLE
ANC Logic - diagnosis Syphilis

### PR DESCRIPTION
**Pull request by anvmn**
#141

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2022-03-09 21-27

**Comment by anvmn** [comment]
### Next step for Syphilis:
Selection_576


### Next step for NeuroSyphilis:

Selection_577

**Comment by anvmn** [comment]
@mariano-dagostino please review

File: client/src/elm/Pages/Prenatal/RecurrentActivity/View.elm [hunk 1]
```diff
@@ -715,6 +765,74 @@ viewNextStepsContent language currentDate assembled data =
...
+    div [ class "ui form recommended-treatment" ]
+        [ viewCustomLabel language Translate.SyphilisRecommendedTreatmentHeader "." "instructions"
+        , h2 []
+            [ text <| translate language Translate.ActionsToTake ++ ":" ]
+        , div [ class "instructions" ]
+            [ viewInstructionsLabel "icon-pills" (text <| translate language Translate.SyphilisRecommendedTreatmentHelper ++ ".")
+            , p [ class "instructions-warning" ] [ text <| translate language Translate.SyphilisRecommendedTreatmentInstructions ++ "." ]
+            ]
+        , viewCheckBoxSelectCustomInput language
+            [ TreatementPenecilin1
```
Comment (mariano-dagostino) [inline client/src/elm/Pages/Prenatal/RecurrentActivity/View.elm]: @anvmn can we use `allowedRecommendedTreatmentSigns`  here?

**Review by mariano-dagostino — APPROVED** [review]
@anvmn looks good. One small potential improvement

**Pull request merged**